Birdui/Birdui/PostView.swift

```diff
@@ -9,20 +9,44 @@
 import SwiftUI
 
 struct PostView: View {
-  let post: MediaPost
-  
-  var body: some View {
-    // TODO: This should look exactly like Birdie's table view cell.
-    // The post text is left-aligned below the mascot image.
-    // The image, if any, is horizontally centered in the view.
-    Text("Layout the view for each post.")
-  }
+    let post: MediaPost
+    
+    var body: some View {
+        // TODO: This should look exactly like Birdie's table view cell.
+        // The post text is left-aligned below the mascot image.
+        // The image, if any, is horizontally centered in the view.
+        
+        VStack{
+            HStack {
+                Image("mascot_swift-badge")
+                    .resizable()
+                    .frame(width: 50, height: 50)
+                VStack(alignment: .leading) {
+                    Text("\(post.userName)")
+                    //temp untill I figure out how to format it properly
```

Comment:
I think you already figured out the formatting issue. Now you can remove this comment. 🥳